plugins/microsoft_teams/unit_test/test_send_message.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+
+from unittest import TestCase
+from icon_microsoft_teams.actions.send_message import SendMessage
+from icon_microsoft_teams.actions.send_message.schema import Input, Output
+from unit_test.util import Util
+from unittest.mock import patch
+from parameterized import parameterized
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+sys.path.append(os.path.abspath("../"))
```

Comment:
This sys path append must be before local imports. For the same reason as in [MC-704] PR